swiss/src/app/landsraad/landsraad.component.ts

```diff
@@ -93,6 +164,7 @@ export class LandsraadComponent implements OnInit {
         }
       )
     }
+    // TODO clear out the form for new question
```

Comment:
You plan this for next PR?

Reply:
I thought it would be nice to have. But I will remove it. I actually removed it already but did not push. Can be done in the future if we figure out it is something we want to have.